composer.json

```diff
@@ -15,7 +15,7 @@
         "forum": "https://discourse.laminas.dev/"
     },
     "require": {
-        "php": "^7.3 || ~8.0.0",
+        "php": "~7.4 || ~8.0.0 || ~8.1.0",
```

Comment:
We should probably have a `composer.lock`

Reply:
should we run composer update prefer lowest for it?

Reply:
Need to first try it out - I think lowest deps may have incompatibilities with 8.1

Reply:
Yes, it seems, I commited composer.lock with prefer-stable